File: HoWo5_6/PhoneCompany/Huawei.java
package HoWo5_6.PhoneCompany;

import HoWo5_6.Phone.Smartphone;

public class Huawei extends Smartphone{

    public Huawei(String model, double sizeDisplay, int akb, String typeOfAntenna, String osVerion, int price) {
        super(model, sizeDisplay, akb, price, ("Android " + osVerion) , typeOfAntenna);
    }

    @Override
    public String toString() {
        return "Huawei "+ super.toString();
    }

}

File: HoWo7/PhoneCompany/Huawei.py
from Phone.Smartphone import Smartphone;

class Huawei(Smartphone):
    def __init__(self,model:str, sizeDisplay, akb: int, typeOfAntenna:str, osVerion:str, price:int):
        super().__init__(model, sizeDisplay, akb, price, ("Android " + osVerion) , typeOfAntenna)

    def __str__(self) ->str:
        return "Huawei " + super().__str__()